core/src/main/scala/org/apache/spark/sql/SnappyDDLParser.scala

```diff
@@ -291,6 +307,68 @@ abstract class SnappyDDLParser(session: SparkSession)
     }
   }
 
+  protected final def policyFor: Rule1[String] = rule {
+    (FOR ~ capture(ALL | SELECT | UPDATE | INSERT | DELETE)).? ~> ((forOpt: Any) =>
+      forOpt match {
+        case Some(v) => v.asInstanceOf[String].trim
+        case None => SnappyParserConsts.SELECT.upper
+      })
+  }
+
+  protected final def policyTo: Rule1[Seq[String]] = rule {
+    (TO ~
+        (CURRENT ~ '_' ~ USER ~ push(CURRENT_USER)  |
+            (LDAPGROUP ~ ws ~ ':' ~ ws ~
```

Comment:
Might be cleaner to make this consistent with LDAPGROUP: make CURRENT_USER as a nonReservedKeyword and use that instead of using '_' this way.

Reply:
OK. will do that

Reply:
Actually this CURRENT_USER is ambiguous because there is added a  CURRENT_USER function too.  Dae Song got confused. I had mentioned this in the call . It would have been better if they were different.